Packs/CommonTypes/IndicatorTypes/reputation-ip.json

```diff
@@ -254,7 +254,7 @@
         "asowner": {
             "simple": "",
             "complex": {
-                "root": "URL",
+                "root": "IP",
```

Comment:
@Ni-Knight Are you familiar with this incorrect mapping?

Reply:
Yes, he talked to me, I approved the PR already :)